workflow-service/src/main/java/com/redhat/parodos/workflow/definition/dto/WorkDefinitionResponseDTO.java

```diff
@@ -61,12 +63,15 @@ public class WorkDefinitionResponseDTO {
 
 	private String author;
 
-	private List<WorkDefinitionResponseDTO> works;
+	private Set<WorkDefinitionResponseDTO> works;
 
 	private Map<String, Map<String, Object>> parameters;
 
 	private List<WorkFlowTaskOutput> outputs;
 
+	@JsonIgnore
```

Comment:
I don't think we need to ignore it

Reply:
ignore when empty might be better?

Reply:
This `checkerId` is the `WorkFlowCheckerMappingDefinition` ID so it will not be of any use for the user in the end as the CHECKER will be in the `works` of the `WorkDefinition`

Plus, the value of `checkerId` will not match the ID of the CHECKER:
```
{
                "id": "a8e50582-6aa9-45ea-b7a2-2195bb480c92",
                "name": "namespaceWorkFlowTask",
                "workType": "TASK",
                "works": [
                  {
                    "id": "8591dfa2-10de-4940-bcd3-1016d145d2bb",
                    "name": "namespaceApprovalWorkFlowChecker",
                    "workType": "CHECKER",
                    "processingType": "SEQUENTIAL",
                    "works": [
                      {
                        "id": "7aa1bd2b-614b-41ea-a3db-654a32363da2",
                        "name": "namespaceApprovalWorkFlowCheckerTask",
                        "workType": "TASK"
                      }
                    ]
                  }
                ],
                "parameters": {
                  "projectId": {
                    "description": "The project id",
                    "type": "number",
                    "required": true
                  }
                },
                "outputs": [
                  "HTTP2XX"
                ],
                "checkerWorkId": "bde50387-7387-4d7b-9ca7-d242b172f8e5"
```
As the ID of the checker is the ID of the WorkfFlow of type CHECKER

Anyway, I'll rename it to `workFlowCheckerMappingDefinitionId` to be more accurate and clear